notebooks/notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>In general some in line comments can improve the readability and clarity of the section.</p><p> </p><p>Consider merging some code chunks specially if there is not explanatory markdown text in between.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/repro-challenge-team-3/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>This code chunk has been merged into a script. We have also added some markdown text explaining what this script does.</p>

Reply:
closing conversation, resolved